client/my-sites/checkout/composite-checkout/hooks/use-get-thank-you-url/get-thank-you-page-url.ts

```diff
@@ -291,7 +292,8 @@ function maybeShowPlanBumpOffer( {
 		return;
 	}
 	if ( hasPremiumPlan( cart ) && ! isTransactionResultEmpty && ! didPurchaseFail ) {
-		return `/checkout/${ siteSlug }/offer-plan-upgrade/business/${ pendingOrReceiptId }`;
+		const upgradeItem = hasMonthlyCartItem( cart ) ? 'business-monthly' : 'business';
+		return `/checkout/${ siteSlug }/offer-plan-upgrade/${ upgradeItem }/${ pendingOrReceiptId }`;
```

Comment:
Could you add a test for this in client/my-sites/checkout/composite-checkout/test/composite-checkout-thank-you.js?

Reply:
Hey @sirbrillig, I have added the test in 93d9b6b.

Reply:
@sirbrillig Since I am aware that you are at a virtual meetup so I've gone ahead and approved this. If you find anyplace that we can improve, please let us know and we can enhance as follow-ups 🙇

Reply:
Looks good to me. Thank you for adding that test!